pkg/cloud/ibmcloud/actuators/machine/actuator.go

```diff
@@ -160,31 +159,32 @@ func (ic *IbmCloudClient) Delete(ctx context.Context, cluster *clusterv1.Cluster
 
 // Update updates a machine and is invoked by the Machine Controller
 func (ic *IbmCloudClient) Update(ctx context.Context, cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
-	log.Printf("Updating machine %v for cluster %v.", machine.Name, cluster.Name)
+	klog.Infof("Updating machine %v for cluster %v.", machine.Name, cluster.Name)
+
 	klog.Infof("TODO: Not yet implemented")
 	return nil
 }
 
 // Exists test for the existance of a machine and is invoked by the Machine Controller
 func (ic *IbmCloudClient) Exists(ctx context.Context, cluster *clusterv1.Cluster, machine *clusterv1.Machine) (bool, error) {
-	guest, err := ic.guestExists(machine)
+	_, err := ic.guestExists(machine)
```

Comment:
I assume the function will return NOTFOUND error in err, right?

Reply:
This function only returns bool. It does not need `guest`. And `guestExists` always return `error` if there is no host found